modules/custom/openy_system/src/Form/OpenyModulesListForm.php

```diff
@@ -91,7 +91,7 @@ protected function buildPackageRow(array $package) {
     $row['#requires'] = [];
     $row['#required_by'] = [];
 
-    $modules = system_rebuild_module_data();
+    $modules = \Drupal::service('extension.list.module')->reset()->getList();
```

Comment:
Let's use DI here

Reply:
Ok @hamrant . I'll work on this next week - form has no ContainerInterface implementation in order to pass DI